### ISSUE TITLE
[Feature request] yarn 2 support

### ISSUE THREAD

**Issue by Reon90** [christopherthielen/check-peer-dependencies#10]
`check-peer-dependencies --yarn --install`
Usage Error: Couldn't find a script named "upgrade".

**Comment by christopherthielen** [comment]
I don't think this library will ever truly support yarn 2 because it relies on node_module traversals.  I have a branch that detects yarn2 and uses `yarn up` instead of `yarn upgrade` but I'm sure this will only work if you use node_modules resolver in yarn 2.   Try it out, and if it works I'll merge #40

**Comment by Chaz0r** [comment]
Fyi: yarn up should work. I tried this package on yarn 2 and I use pnp. I got the error that yarn upgrade does not exist. So i simply added a script in my package json called "upgrade" which simply runs "yarn up". 
In that case yarn upgrade exists and executes yarn up. Worked perfectly fine.
